sdk/tests/invariant.test.ts

```diff
@@ -55,4 +56,83 @@ describe('invariant', function () {
 
     assert.deepEqual(newFee, { v: 100 })
   })
+
+  it('should add fee tier', async () => {
```

Comment:
Have you tried fitting `beforeEach` hook to this pattern? I mean, returning the created invariant contract and everything before it